src/client.ts

```diff
@@ -451,4 +454,28 @@ export class FarosClient {
       );
     }
   }
+
+  async getWebhookEvent(
+    webhookId: string,
+    eventId: string
+  ): Promise<WebhookEvent> {
+    try {
+      const response = await this.api.get(
+        `/webhooks/${webhookId}/events/${eventId}`
+      );
+      let event = response.data;
+      event = {
+        ...event,
+        createdAt: event.createdAt ? new Date(event.createdAt) : undefined,
```

Comment:
@JoshSauter how come `createdAt` can be undefined but all other timestamp fields not?

Reply:
also, you can simply use `Utils.toDate(event.createdAt)` here and below

Reply:
ohh, `createdAt` is a timestamp from the source system which might not be necessarily set.